From 59098696fc10e36f429fb71d999b69b72fe87564 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Tue, 18 Oct 2022 10:11:48 +0300
Subject: [PATCH] [SW-0] gaudi3: fuse: set sku value explicitly

Since sku value stored in ROM global data structure which is
persistent memory, this patch set sku value explicitly as part of
fuse init routine to aviod stale value across reset.

Change-Id: I7a015c449781004b3ee4ee39248f4f66da5796a7
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index 166c811..2ebabf4 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -618,6 +618,8 @@
 
 	if (arcboot_rom_fuse_single_die_config())
 		arcboot_rom_global.sku = 1;
+	else
+		arcboot_rom_global.sku = 0;
 
 	/* Override the fuse values as per requirement */
 	arcboot_rom_fuse_override();
